src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -144,42 +148,36 @@ enum SideMenuButtonType {
         }
 
         let backLabel = setupLabel(title: kLocalizedBack, selector: #selector(delegate?.stopAction), type: .landscapeFirstColumn)
-        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor, constant: marginLabel).isActive = true
+        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor).isActive = true
 
-        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        if #available(iOS 11.0, *) {
-            continueButton.topAnchor.constraint(equalTo: self.safeAreaLayoutGuide.topAnchor, constant: marginTopBottom).isActive = true
-        } else {
-            continueButton.topAnchor.constraint(equalTo: self.topAnchor, constant: marginTopBottom).isActive = true
-        }
+        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueButton.topAnchor.constraint(equalTo: backLabel.bottomAnchor, constant: marginTopBottom * 2).isActive = true
 
-        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor, constant: marginLabel).isActive = true
+        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor).isActive = true
+        continueLabel.bottomAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
```

Comment:
Please set the continueLabel only to the Mid Anchor and the ContinueButton to the Top of the Continue Label, so it does not resize on smaller phones

Reply:
![Bildschirmfoto 2021-04-09 um 11 46 35](https://user-images.githubusercontent.com/29690863/114165423-c228da00-992c-11eb-94af-4abb83217a35.png)